src/chainstate/coordinator/tests.rs

```diff
@@ -291,8 +292,9 @@ pub fn make_coordinator<'a>(
     path: &str,
     burnchain: Option<Burnchain>,
 ) -> ChainsCoordinator<'a, NullEventDispatcher, (), OnChainRewardSetProvider> {
+    let (tx, _) = sync_channel(1000);
```

Comment:
Is 1000 enough?  It might make sense to make this unbound, or to implement some kind of buffering system outside the channel.  The reason is that when a node is bootstrapping and processing lots and lots of transactions in a short timespan, it may be possible to saturate this channel.

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2013/commits/31aa6589e0a731e2c1ca1bac51c46d0d1ef91c74